src/components/AppProvider/utilities/createPolarisContext/createPolarisContext.ts

```diff
@@ -42,5 +42,5 @@ export default function createPolarisContext(
     ? createThemeContext(themeContext)
     : createThemeContext();
 
-  return {...appProvider, ...theme};
+  return {...appProvider, theme};
```

Comment:
Theme is now `{logo, ...}`. We'll want to namespace all the theme properties under `theme`.

Reply:
Can you explain this a bit deeper? Not sure I understand

Reply:
`theme` was previously `{polarisTheme: {logo: null}}` and now it's `{logo: null}`. So previsouly this would have returned
```tsx
{...appProvider, polarisTheme: {logo: null}};
```
and now returns 
```tsx
{...appProvider, theme: {logo: null)};
```

Reply:
If we still spread `theme` the result would be
```tsx
{...appProvider, logo: null};
```